src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsLookAndFeel.java

```diff
@@ -657,7 +658,8 @@ protected void initComponentDefaults(UIDefaults table)
                   new WindowsDesktopProperty("win.caret.width", null),
 
             "CheckBox.font", ControlFont,
-            "CheckBox.interiorBackground", WindowBackgroundColor,
+            "CheckBox.interiorBackground", (xp != null)?
```

Comment:
Do we return any color other than the default FFFFFF/null?
What is the use of "win.frame.backgroundColor" property when we always return table.get("window")?

Reply:
`win.frame.backgroundColor` is key and value is color set for `window`. Default color is FFFFF/null, when it is set by user the color updates.

Reply:
From WindowsDesktopProperty.java constructor it looks like these are not key value pairs. Similar observation is there in DesktopProperty.java
It looks like table.get("window") is fallback when win.frame.backgroundColor is null.

Reply:
Ok, but we are not modifying `win.frame.backgroundColor` property, only` "CheckBox.interiorBackground"` is been set to null. `WindowBackgroundColor` is also used in other properties, so nothing is modified except ` "CheckBox.interiorBackground"`.